go.mod

```diff
@@ -59,6 +64,7 @@ require (
 	github.com/dgryski/go-rendezvous v0.0.0-20200823014737-9f7001d12a5f // indirect
 	github.com/golang/protobuf v1.5.2 // indirect
 	github.com/golang/snappy v0.0.0-20180518054509-2e65f85255db // indirect
+	github.com/letsencrypt/validator/v10 v10.0.0-20230215210743-a0c7dfc17158
```

Comment:
Seems like this should be in the top-most stanza, next to all our other direct dependencies.